README.md

```diff
@@ -138,35 +138,72 @@ oc policy add-role-to-user edit system:serviceaccount:<your-namespace>:miq-orche
+* Database (PostgreSQL, required if running database podified)
 
-Example NFS PV templates are provided, **please skip this step you have already configured persistent storage.**
+NFS PV templates are provided, **please skip this step you have
+already configured persistent storage.**
 
-For NFS backed volumes, please ensure your NFS server firewall is configured to allow traffic on port 2049 (TCP) from the OpenShift cluster.
+For NFS backed volumes, please ensure your NFS server firewall is
+configured to allow traffic on port 2049 (TCP) from the OpenShift
+cluster.
 
 _**Note:**_ Recommended permissions for the PV volumes are 777, root uid/gid owned.
 
 _**As admin:**_
 
-Please inspect example NFS PV files and edit settings to match your site. You will at a minimum need to configure the correct NFS server host and appropiate path.
+Creating the required PVs may be a one or two step process. You may
+create the initial *templates* now, and then process them and create
+the *PV*s later, or you may do all of the processing and PV creation
+in one pass
 
-Create PV
-```bash
-$ oc create -f templates/miq-pv-db-example.yaml
+There are three parameters required to process the templates. Only
+`NFS_HOST` is required, `PV_SIZE` and `BASE_PATH` have sane defaults
+already
+
+* `PV_SIZE` - **Defaults** to the recommended PV size for the App/DB
+  template (`5Gi`/`15Gi` respectively)
+* `BASE_PATH` - **Defaults** to `/exports`
+* `NFS_HOST` - **No Default** - Hostname or IP address of the NFS
+  server
+
+_**Note**_ Repeat this process for `templates/miq-pv-db-example.yaml`
+as well if you are running your PostgreSQL database podified. The name
+of the database pv template is `manageiq-db-pv`.
+
+#### Method 1 - Create Template, Process and Create Later
+
+This method first creates the template object in OpenShift and then
+demonstrates how to process the template and fill in the required
+parameters at a later time.
+
+```
 $ oc create -f templates/miq-pv-server-example.yaml
+# ... do stuff ...
+$ oc process manageiq-app-pv -p NFS_HOST=nfs.example.com | oc create -f -
+```
+
+
+
+#### Method 2 - Process Template and Create PV in one pass
+
+```
+# oc process templates/miq-pv-server-example.yaml -p NFS_HOST=nfs.example.com | oc create -f -
 ```
+
 Verify PV creation
+
 ```bash
 $ oc get pv
-NAME       CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS      CLAIM  REASON   AGE
-miq-pv01   15Gi        RWO           Recycle         Available                   30s
-miq-pv02   5Gi         RWO           Recycle         Available                   19s
+NAME      CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS    CLAIM                                       STORAGECLASS   REASON    AGE
+miq-app   4Gi        RWO           Retain          Bound     manageiq/manageiq-server-manageiq-0                                  1d
+miq-db    10Gi       RWO           Retain          Bound     manageiq/manageiq-postgresql                                         1d
```

Comment:
After PV creation, but before deploying the app I wouldn't expect these to be `Bound`